### ISSUE TITLE
Structuring the repo

### ISSUE THREAD

**Issue by sakshamj74** [sakshamj74/IPL-Analysis#103]
Like at this present, we have multiple files and they are not organized and feel a little messy. The end goal is to structure the repo so it becomes easy to contribute and to keep track of what has been done.

**Comment by AdityaJ7** [comment]
Hi, can I take this up?

**Comment by sakshamj74** [comment]
@AdityaJ7 Yes you can. But please give me some idea of how you will be structuring this repo?
